build.gradle

```diff
@@ -232,6 +232,11 @@ dependencyManagement {
             entry 'tomcat-embed-el'
             entry 'tomcat-embed-websocket'
         }
+
+        // CVE-2019-10086
+        dependencySet(group: 'commons-beanutils', version: '1.9.4') {
```

Comment:
it is currently fixed in master, we had to upgrade checkstyle plugin version to use the latest commons-beanutils
please remove that set, I think it is not needed now